src/spdigger_stack.c

```diff
@@ -2314,7 +2314,7 @@ long check_out_uncrowded_reinforce_position(struct Thing *thing, SubtlCodedCoord
                     pos.x.val = subtile_coord_center(stl_x);
                     pos.y.val = subtile_coord_center(stl_y);
                     pos.z.val = get_thing_height_at(thing, &pos);
-                    if ( creature_can_navigate_to_with_storage(thing, &pos, NavRtF_Default) != -1 ) {
```

Comment:
I indeed never see it be -1, but pretty unrelated to cmake I think.

Reply:
Just fix the warning reported by the new compilers.

Reply:
That function returns TbBool, so checking with != -1 should be incorrect.